src/EmotiBitSerial.cpp

```diff
@@ -0,0 +1,35 @@
+#include "EmotiBitSerial.h"
+
+#ifdef ARDUINO
+bool EmotiBitSerial::parseSerialMessage(String message, String &typetag, String &payload)
+{
+	if(message.charAt(0) == MSG_START_CHAR)
+	{
+		int payloadDelimiterIndex = message.indexOf(PAYLOAD_DELIMITER);
+		int messageDelimiterIndex = message.indexOf(MSG_TERM_CHAR);
+		if (payloadDelimiterIndex > 0 && messageDelimiterIndex > 0)
```

Comment:
I was iffy about implementing the +2 (for the empty string test). But, after thinking about it, I added it.
Reason being, if we have 1 space, it can be unintentional, but if its more than a space, the user probably meant it.

So I added both checks.